apps/user-office-backend/src/auth/StfcUserAuthorization.ts

```diff
@@ -242,22 +242,22 @@ export class StfcUserAuthorization extends UserAuthorization {
           return Promise.resolve('User already logged out');
         }
 
-        return await client
-          .logout(token)
-          .catch((ex: string): Rejection | string => {
-            if (ex === 'The token given is invalid') {
-              logger.logInfo(
-                'Logout failed because the cached token has since expired',
-                { token }
-              );
-
-              return 'User already logged out';
-            }
-
-            logger.logWarn('Failed to log out user', { token });
-
-            return rejection('Failed to log out user', { token });
-          });
+        await client.logout(token).catch((ex: string): Rejection | string => {
+          if (ex === 'The token given is invalid') {
+            logger.logInfo(
+              'Logout failed because the cached token has since expired',
+              { token }
+            );
+
+            return 'User already logged out';
+          }
+
+          logger.logWarn('Failed to log out user', { token });
+
+          return rejection('Failed to log out user', { token });
+        });
+
+        return 'Successfully logged out user';

```

Comment:
Is it possible to do it like this
```
        return await client.logout(token).then(()=>{
          return 'Successfully logged out user';
        }).catch((ex: string): Rejection | string => {
          if (ex === 'The token given is invalid') {
            logger.logInfo(
              'Logout failed because the cached token has since expired',
              { token }
            );

            return 'User already logged out';
          }

          logger.logWarn('Failed to log out user', { token });

          return rejection('Failed to log out user', { token });
        });
```